k4EDM4hep2LcioConv/src/k4EDM4hep2LcioConv.cpp

```diff
@@ -27,16 +27,16 @@ lcio::LCCollectionVec* convTracks(
       lcio_tr->setRadiusOfInnermostHit(edm_tr.getRadiusOfInnermostHit());
 
       // Loop over the hit Numbers in the track
-      lcio_tr->subdetectorHitNumbers().resize(edm_tr.subDetectorHitNumbers_size());
```

Comment:
Yes. good point. The problem is that only the current master branch of edm4hep ships something that is easily usable here, so I might have to do some "bootstrapping" to make this works somewhat transparently. I will have a look.

Reply:
```
#ifdef EDM4HEP_VERSION
#DEFINE SUBDETECTOR subdetector
...
```
?

Reply:
Thanks!

Reply:
The only problem now is that immediately after key4hep/EDM4hep#188 is merged this will fail again, because the `EDM4HEP_BUILD_VERSION` will only change if we have a new tag.